src/data/list/defs.lean

```diff
@@ -96,6 +96,16 @@ def take_while (p : α → Prop) [decidable_pred p] : list α → list α
 | []     := []
 | (a::l) := if p a then a :: take_while l else []
 
+/-- `after p xs` is the suffix of `xs` after the first element that satisfies
+  `p`, not including that element.
+
+     after      (eq 1)       [0, 1, 2, 3] = [2, 3]
```

Comment:
These lines should be in a code block
````
```lean
after ...
```
````

Reply:
Done (though none of the other defs in that module seem to adhere to this convention).

Reply:
Oh! Somehow I didn't notice the indentation, sorry. That's equivalent to triple backticks in most flavors of markdown.

Reply:
I see. The backticks are better anyway as far as I'm concerned, so I'll leave it as is.